src/bin.ts

```diff
@@ -326,6 +331,8 @@ export function main(
   if (executeEntrypoint) {
     Module.runMain();
   } else {
+    // Note: eval and repl may both run, but never with stdin.
```

Comment:
If that's the case, I think the `if` branching should reflect that, to make it clear. ie:

```
if (!executeStdin) {
  if (executeEval) { ... }
  if (executeRepl) { ... }
} else {
 ..
}
```